subsys/net/lib/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -214,3 +214,49 @@
 	OPENTHREAD_CONFIG_PLAT_LOG_FUNCTION__COUNT_ARGS
 
 #endif  /* OPENTHREAD_CORE_NRF52840_CONFIG_H_ */
+
+/**
+ * @def OPENTHREAD_CONFIG_RADIO_LINK_IEEE_802_15_4_ENABLE
+ *
+ * Set to 1 to enable support for IEEE802.15.4 radio link..
+ *
+ */
+#ifdef CONFIG_OPENTHREAD_RADIO_LINK_IEEE_802_15_4_ENABLE
```

Comment:
Why aren't the new symbols defined in Kconfig file(s)?

Reply:
Yeah, I missed that file. Fixed.